docs/manuals/core/versioning/configuration.md

```diff
@@ -51,10 +51,10 @@ the [Advanced configuration page](../config/advanced-config.md).
 
 The versioning management system has many parameters that you can modify to accommodate your
 use-case (such as development or experiment mode). To see a list of all predefined Taipy options,
-you can run any Taipy script that runs a `Core` instance with the *-h* or *--help* option.
+you can run the `taipy help run` command. Alternatively, you can use the `--help` or `-h` options
```

Comment:
Please don't use fixed-width font for thigs that are not commands or API elements.
\*-h\* what fine in my opinion.